### DISCUSSION TITLE
French language support

### DISCUSSION THREAD

**Discussion by blamaz** [moralmunky/Home-Assistant-Mail-And-Packages#737] [Ideas]
Hi,
Is there a plan to add more supported languages.
I would like to use this addon in french.
Thank you for this excellent package

**Comment by moralmunky** [comment]
There is a [French translation file](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/blob/master/custom_components/mail_and_packages/translations/fr.json).

**Comment by firstof9** [comment]
If you know the keywords from the shippers you can submit a PR as easy as modifying the consts.py file:

https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/blob/f82d1c902c82c935530d4403f741cbf528a0f734/custom_components/mail_and_packages/const.py#L144-L405

**Comment by blamaz** [comment]
Thank you both of you for your help, I will try your suggestions.
